test/unit_tests/braket/experimental/autoqasm/test_operators.py

```diff
@@ -363,6 +363,96 @@ def test_logical_eq_qasm_cond() -> None:
     assert "==" in qasm
 
 
+def test_logical_ops_qasm() -> None:
+    """Tests the logical aq.operators for QASM expressions."""
+
+    @aq.subroutine
+    def do_and(a: bool, b: bool):
```

Comment:
Unit tests should try to test only one thing. I can see why you'd want to put this all into one big test, to avoid all the boilerplate code. You could instead make a testing function to take in the subroutine and an expected string

Reply:
Yeah, I thought you were avoiding making multiple tests because it might introduce boilerplate

Reply:
Thank you :)